--- Makefile~	2009-09-07 16:17:33.000000000 +0200
+++ Makefile	2009-09-07 16:18:30.000000000 +0200
@@ -36,7 +36,7 @@
 CLTHREADS_SO = libclthreads.so
 CLTHREADS_MAJ = $(CLTHREADS_SO).$(MAJVERS)
 CLTHREADS_MIN = $(CLTHREADS_MAJ).$(MINVERS)
-CLTHREADS_DEP = -lpthread
+CLTHREADS_DEP = -lpthread -lrt
 CLTHREADS_O = p_thread.o a_thread.o itc_mesg.o itc_ip1q.o itc_ctrl.o textmsg.o
 CLTHREADS_H = clthreads.h
 
